components/infoColumns/index.tsx

```diff
@@ -60,31 +59,32 @@ const CardContainer = styled.div<{
   ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
 `;
 const Title = styled(SubTitle)`
-  font-size: 4rem;
-  ${breakpoints("font-size", "", [{ 1000: "2.5rem" }])};
+  font-size: 6rem;
+  ${breakpoints("font-size", "", [{ 1000: "2.25rem" }])};
   width: 100px;
   ${breakpoints("width", "", [{ 1000: "100%" }])};
-  margin: 0;
+  margin: 0 0 0 0;
+  font-weight: 1000;
 `;
 
 const InfoColumns = () => {
   return (
     <InfoColumnWrapper>
-      <CardContainer flex="1" margins="1rem 0" pad="1rem 1rem 1rem 0">
-        <Title>About CIMUN</Title>
-        <Body size="1rem" line={1.4}>
+      <CardContainer flex="1" margins="1rem 0" pad="0 2rem 0 0">
+        <Title line={0.9}>Welcome Back, Delegates.</Title>
+        <Body line={1.4} align={"justify"}>
```

Comment:
Align=justify wasn't rly the issue it still has the 2rem padding ![image](https://user-images.githubusercontent.com/55511406/124227481-bd616580-dad8-11eb-82e2-b3a39503db71.png)